doc/specs/#653 - Quake Mode/#653 - Quake Mode.md

```diff
@@ -0,0 +1,680 @@
+<td>Move the window to this desktop (leave position alone)</td>
+<td>
+
+If there isn't one on this desktop:
+* create a new one (default position)
+
+Else:
+* activate the one on this desktop (don't move it)
+</td>
+</tr>
+<!-- ----------------------------------------------------------------------- -->
+<tr>
+<td><code>"toCurrent"</code><br> Summon the MRU window TO the current monitor</td>
+<td>Go to the desktop the window is on, move to this monitor</td>
+<td>Move the window to this desktop, move to this monitor</td>
+<td>
+
+If there isn't one on this desktop:
+* create a new one (on this monitor)
+
+Else:
+* activate the one on this desktop, move to this window
+</td>
+</tr>
+<!-- ----------------------------------------------------------------------- -->
+<tr>
+<td><code>"onCurrent"</code><br> Summon the MRU window for the current monitor</td>
+<td>
+
+If there is a window on this monitor on any desktop,
+* Go to the desktop the window is on (leave position alone)
+
+else
+* Create a new window on this monitor  & desktop
+</td>
+<td>
+
+If there is a window on this monitor on any desktop,
+* Move the window to this desktop (leave position alone)
+
+else
+* Create a new window on this monitor  & desktop
+</td>
+<td>
+
+If there isn't one on this desktop, (even if there is one on this monitor on
+another desktop),
+* create a new one on this monitor
+
+Else if ( there is one on this desktop, not this monitor)
+* create a new one on this monitor
+
+Else (one on this desktop & monitor)
+* Activate the one on this desktop (don't move)
+</td>
+</tr>
+<!-- ----------------------------------------------------------------------- -->
+<tr>
+<td><code>int</code><br> Summon the MRU window for monitor N</td>
+<td>
+
+If there is a window on monitor N on any desktop,
+* Go to the desktop the window is on (leave position alone)
+
+else
+* Create a new window on this monitor  & desktop
+</td>
+<td>
+
+If there is a window on monitor N on any desktop,
+* Move the window to this desktop (leave position alone)
+
+else
+* Create a new window on this monitor & desktop
+</td>
+<td>
+
+If there isn't one on this desktop, (even if there is one on monitor N on
+another desktop),
+* create a new one on monitor N
+
+Else if ( there is one on this desktop, not monitor N)
+* create a new one on monitor N
+
+Else (one on this desktop & monitor N)
+* Activate the one on this desktop (don't move)
+</td>
+</tr>
+</table>
+
+
+##### Stories, revisited
+
+With the above settings, let's re-examine the original user stories, and see how
+they fit into the above settings. (_Stories that are omitted aren't relevant to
+the discussion of these settings_)
+
+> When the `desktop` param is omitted below, that can be interpreted as "any
+> `desktop` value will make sense here"
+
+* **Story A** Press a hotkey anywhere to activate the single Terminal window
+  wherever it was
+  - This is `{ "monitor": "any", "desktop": "any" }`

```

Comment:
I really think you should offer some sort of short-hand option that does this on the back end. I understand by reading this spec why they matrix exists and how it offers the ultimate in flexibility. But I also believe that the consumer of some of these stories are going to have their eyes utterly glaze over at how to convert what they want to do into the correct sequence of events down here. 

Then you could choose the { "summonEzMode" : "activateWherever"} or {"summonEzMode": "activateInMyFace"}  to further simplify some of "simple" scenarios that become complex.

Alternatively or in addition, I think we'll need both a docs.microsoft page showing a ton of these examples mapping the plain language behavior to the JSON, an EZ-mode sort of scenario drop down in the Settings UI that applies this behavior on the back end, and perhaps some of these just commented out and hanging out in the JSON file. Especially the "additional examples" below... they get a bit hairy in terms of how you specify them.

Reply:
I don't hate that. I'll stick that in the future considerations. I'm keeping the names 😝